examples/wordpress/frontend.jsonnet

```diff
@@ -0,0 +1,100 @@
+local kube = import "lib/kube.libsonnet";
+
+local labels = {
+  tier: "frontend",
+};
+
+{
+  frontend_pvc: kube.PersistentVolumeClaim("wordpress") {
+    metadata+: {
+      labels+: labels,
+    },
+    storage:: "10Gi",
+  },
+
+  frontend_configmap: kube.ConfigMap("wordpress") {
+    metadata+: {
+      labels+: labels,
+    },
+    data: {
+      "admin_first_name": "Admin",
+      "admin_last_name": "User",
+      "blog_name": "Kubernetes blog!",
+  }},
+
+  frontend_secret: kube.Secret("wordpress") {
+    metadata+: {
+      labels+: labels,
+    },
+    data_+: {
+      "user": "user",
+      "password": "bitnami",
+      "mail": "user@example.com",
+  }},
+
+  frontend_deployment: kube.Deployment("wordpress") {
+    metadata+: {
+      labels+: labels,
+    },
+    spec+: {
+      template+: {
+        spec+: {
+          containers_+: {
+            default: kube.Container("wordpress") {
+              image: "bitnami/wordpress",
+              ports_+: { http: { containerPort: 80 } },
+	      env_+: {
```

Comment:
Incorrectly indented (and the next line)

Reply:
Fixed